projects/Mallard/src/hooks/use-cached.ts

```diff
@@ -0,0 +1,63 @@
+import NetInfo from '@react-native-community/netinfo'
+
+import { useEffect, useState } from 'react'
+import { useSettings } from './use-settings'
+import axios from 'axios'
+import RNFetchBlob from 'rn-fetch-blob'
+import { DatePickerAndroid } from 'react-native'
+
+export const useCached = <Fetchable>(
+    path: string,
+    initialState: Fetchable,
+): Fetchable => {
+    const [{ apiUrl }] = useSettings()
+    const localPath = `storage/${path}`
```

Comment:
this is temporary as we want this to eventually be the same as the downloaded issues, right?

Reply:
I just want to see if this works for now.
Will wire up later.